cartridges/app_custom_pd/cartridge/client/default/scss/experience/components/commerceAssets/categoryComponent.scss

```diff
@@ -0,0 +1,68 @@
+@import "~base/variables";
+
+.category-img{
+    object-fit: fill;
+}
+
+.tile-container {
+    position: relative;
+}
+
+.cta-link {
+    position: absolute;
+    left: 50%;
+    transform: translateX(-50%);
+    background-color: rgba(255, 255, 255, 0.5);
+    padding: 0rem 0.4rem;
+    border-radius: 0.2rem;
+    border: 1px solid $black;
+    white-space: nowrap;
+    transition: background-color 0.3s ease, color 0.3s ease;
+}
+
+.cta-link a {
+    text-decoration: none;
+}
+
+.cta-link:hover {
+    background-color: $white;
+}
+
+.bottom {
+    bottom: 0.5rem;
+}
+
+.cat-description {
```

Comment:
Still part of these properties are covered in .cta-link. You can use @extend in order to take those from there.

Reply:
fixed